var/spack/repos/builtin/packages/py-pbr/package.py

```diff
@@ -34,3 +34,4 @@ class PyPbr(PythonPackage):
     version('1.8.1', 'c8f9285e1a4ca6f9654c529b158baa3a')
 
     depends_on('py-setuptools', type='build')
+    depends_on('py-enum34', type='build')
```

Comment:
I added a comment, but that is not particularly helpful -> Please advise on how this should be handled (where can I put a guarding ```if``` clause to protect against that failing silently?)